extensions/notebook-renderers/src/test/notebookRenderer.test.ts

```diff
@@ -8,20 +8,20 @@ import { activate } from '..';
 import { RendererApi } from 'vscode-notebook-renderer';
 import { IDisposable, IRichRenderContext, OutputWithAppend, RenderOptions } from '../rendererTypes';
 import { JSDOM } from "jsdom";
+import { LinkDetector } from '../linkify';
 
 const dom = new JSDOM();
 global.document = dom.window.document;
 
 suite('Notebook builtin output renderer', () => {
 
 	const error = {
-		name: "NameError",
-		message: "name 'x' is not defined",
+		name: "TypeError",
+		message: "Expected type `str`, but received type `<class \'int\'>`",
 		stack: "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m" +
-			"\n\u001b[1;31mNameError\u001b[0m                                 Traceback (most recent call last)" +
-			"\nCell \u001b[1;32mIn[3], line 1\u001b[0m" +
-			"\n\u001b[1;32m----> 1\u001b[0m \u001b[39mprint\u001b[39m(x)" +
-			"\n\n\u001b[1;31mNameError\u001b[0m: name 'x' is not defined"
+			"\u001b[1;31mTypeError\u001b[0m                                 Traceback (most recent call last)" +
+			"\u001b[1;32mc:\\src\\test\\ws1\\testing.py\u001b[0m in \u001b[0;36mline 2\n\u001b[0;32m      <a href='file:///c%3A/src/test/ws1/testing.py?line=34'>35</a>\u001b[0m \u001b[39m# %%\u001b[39;00m\n\u001b[1;32m----> <a href='file:///c%3A/src/test/ws1/testing.py?line=35'>36</a>\u001b[0m \u001b[39mraise\u001b[39;00m \u001b[39mTypeError\u001b[39;00m(\u001b[39m'\u001b[39m\u001b[39merror = f\u001b[39m\u001b[39m\"\u001b[39m\u001b[39mExpected type `str`, but received type `\u001b[39m\u001b[39m{\u001b[39m\u001b[39mtype(name)}`\u001b[39m\u001b[39m\"\u001b[39m\u001b[39m'\u001b[39m)\n" +
```

Comment:
using a stacktrace that contains file links so we can check the rendering of those.